app-backend/db/src/main/scala/util/ProjectCache.scala

```diff
@@ -0,0 +1,63 @@
+package com.rasterfoundry.database.util
+
+import java.net.InetSocketAddress
+
+import com.rasterfoundry.common.{BacksplashConnectionFactory, Config}
+import com.rasterfoundry.datamodel._
+import com.typesafe.scalalogging.LazyLogging
+import doobie.ConnectionIO
+import doobie.implicits.AsyncConnectionIO
+import net.spy.memcached.MemcachedClient
+import scalacache._
+import scalacache.memcached.MemcachedCache
+
+import scala.collection.JavaConverters._
+import scalacache.serialization.circe._
```

Comment:
This is cool, pulling this in means that the values are `json` serialized in the cache - which is nice if we ever want to connect directly and figure out what is stored in the cache. It was useful for debugging.